lib/lexisnexis/request.rb

```diff
@@ -16,13 +16,13 @@ def initialize(attributes)
       @url = build_request_url
     end
 
-    def send
+    def send(response_options: {})
```

Comment:
Since there aren't any other options, we could just make this `dob_year_only` explicitly? But it seems to me like this was just a pass-through so I opted to keep it flexible, in case we would want to separate request options, or maybe even retry options or something in the future

Reply:
could docs be added that says what options are accepted?

Reply:
Good call, added in a93ff48258b7d53eeabab1ab42a05f577a800d7f